spec/Makefile

```diff
@@ -1,6 +1,6 @@
 REPO_DIR=$(shell pwd)
 
-run: download transform format
+run: transform format
```

Comment:
![image](https://user-images.githubusercontent.com/8291514/207290364-ffad5958-8ec9-434d-814d-95b2562ba7e7.png)

Reply:
haha undid the hack. supabase-js version is published now